eng/tools/lint-diff/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@azure-tools/lint-diff",
+  "private": true,
+  "type": "module",
+  "main": "dist/src/lint-diff.js",
+  "bin": {
+    "lint-diff": "cmd/lint-diff.js",
+    "summarize-lint-diff": "cmd/summarize-lint-diff.js"
+  },
+  "scripts": {
+    "build": "tsc --build",
+    "test": "vitest",
+    "test:ci": "vitest run --coverage --reporter=verbose",
+    "clean": "rm -rf dist && rm -rf node_modules"
+  },
+  "engines": {
+    "node": ">= 18.0.0"
+  },
+  "dependencies": {
+    "@azure-tools/openapi-tools-common": "^1.2.2",
+    "@azure/openapi-markdown": "^0.9.4",
```

Comment:
Fine to keep these for now, but we should eliminate the dependencies in the future.

Reply:
For yaml and markdown processing, let's standardize on these (in the future, not this PR):

https://github.com/Azure/azure-rest-api-specs/blob/a39f51d0529655d72c80e1d2b55ed88ce6c98ff7/.github/package.json#L6-L9

Reply:
Added to https://github.com/Azure/azure-rest-api-specs/issues/33302